### DISCUSSION TITLE
How many languages should I add?

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#28] [Q&A] [answered]
How many languages should I add to my project: https://github.com/KendallDoesCoding/languages-around-the-world. Right now there are 3 in the project. I think 5-10 will be good?

**Comment by TechStudent10** [comment]
well if you are going to do around the world then you should do all the languages.

![image](https://user-images.githubusercontent.com/76978184/155219167-541ac134-368d-4d65-b023-d7e303bdf980.png)

precisely 6.5k. good luck

**Comment by KendallDoesCoding** [comment]
Firstly, didn't know so many languages existed.

Secondly, uh no for 1 language it takes like 30 mins of research and
putting together.

**Reply by TechStudent10** [reply]
`(((6.5k * 30) / 60) / 60) / 24 = 2.25694444`

So about 3 days if you remove everything from your schedule. A week if you add that stuff back in. So yeah I understand.

**Comment by TechStudent10** [comment, marked as answer]
I would suggest adding about 50 languages.

**Reply by KendallDoesCoding** [reply]
Ok. That will still take a lot of time, idk if I want to make it a ENTIRE project yet as that info can be found on the web with 15 searches, instead of on one page though still. So the website is useful, but not SO useful.
